meta-openpli/recipes-multimedia/readymedia/readymedia.bb

```diff
@@ -8,12 +8,12 @@ LIC_FILES_CHKSUM = "file://LICENCE.miniupnpd;md5=b0dabf9d8e0f871554e309d62ead8d2
 
 inherit gitpkgv
 
-PV = "1.1.0+git${SRCPV}"
-PKGV = "1.1.0+git${GITPKGV}"
-PR = "r0"
+PV = "1.1.6+git${SRCPV}"
```

Comment:
It's "1.2.2" now ;)

Reply:
The number I wrote in PR isn't the version of minidlna. I didn't know what to write, but found some numbers in git and took it. I only wanted to update it. It takes latest commit, so the recipe doesn't now the version. I am new to bitbake and this build environment. I have now figure out that it is enough to change PR or PV. So in this case since I update PR, no requirement to update PV. However nothing wrong to update both.

Can I keep it as it is or what should I write (I don't think it should be related to version of minidlna, but maybe it can be, but it is confusing)?

Reply:
There's no need for PR, just remove it and keep PV with our version which is now 1.2.2 as AUTOREV will get latest updates.

This recipe needs some changes, MAINTAINER is us not Open PLi, there's no need for external "readymedia.sh" and "minidlna.conf" as we provide them.